src/delivery/templates/kitchen_count_steps.html

```diff
@@ -1,28 +1,28 @@
 {% load i18n %}
 
 <div class="ui four ordered steps">
-  <div class="{% if step == 'orders' %} active{% endif %} step">
+  <a class="{% if step == 'orders' %} active {% else %} completed {% endif %} step" href="{% url 'delivery:order' %}">
     <div class="content">
```

Comment:
Using the completed icon, in  this case, is tricky, because we actually don't know if the other steps have been completed. We are better off not showing that.